appveyor.yml

```diff
@@ -0,0 +1,10 @@
+version: 4.0.0.{build}
+platform:
+    - x64
+install:
+    - SET PATH=C:\Ruby215\bin;%PATH%
+    - gem install bundler --quiet --no-ri --no-rdoc
+    - bundle install --without development
+build: off
+test_script:
+    - bundle exec rspec spec
```

Comment:
The feedback I've gotten is generally disable e-mail notifications, but the ci list is a special case. It'd probably get a lot more volume than it does now though, and I'm not sure who watches it.